template_compiler/definitions.rb

```diff
@@ -69,6 +69,22 @@
       "user_editable_file": "app/server.rb"
     }
   ),
+  Repo.new(
+    course: rust,
+    language: rust,
```

Comment:
The course here shouldn't be `rust`, it should be `redis`.

Reply:
Acknowledged sir